app/views/machine.njk

```diff
@@ -102,6 +83,8 @@
 {% endblock %}
 
 {% block js %}
-{#  <script src="/js/vendor/jquery.tablesort.min.js"></script>#}
-{#  <script src="/js/machine.js"></script>#}
+ <script src="/js/machine.js"></script>
+ <script>
+  generateCalendar(backupCalendar)
```

Comment:
Instead of calling this function here, call it at the end of `public/js/machine.js`. Then, you can delete this `<script>` tag entirely.